### DISCUSSION TITLE
Self-hosted console asks to activate user

### DISCUSSION THREAD

**Discussion by Susheelkaram** [zitadel/zitadel#8878] [Q&A]
I have local zitadel installation setup on my machine. It worked fine on initial setup. It shut down upon shutting down laptop.
I reran instance using command in [setup](https://zitadel.com/docs/self-hosting/deploy/macos) to start zitadel server and upon opening console link, it asked me to activate user.
http://localhost:8080/ui/console

<img width="865" alt="Screenshot 2024-11-10 at 1 26 42 AM" src="https://github.com/user-attachments/assets/6cddd83e-c8c3-4501-8ddb-dfd72bc8a0c2">

**Comment by fforootd** [comment]
Can you share how you started/deployed Zitadel?

**Reply by Susheelkaram** [reply]
Sorry for late reply. I am currently using below with respective config files. It works fine on fresh installation, but stops working after somedays of not using (I am working on a side project).
```
'zitadel start-from-init \
    --config ./example-zitadel-config.yaml \
    --config ./example-zitadel-secrets.yaml \
    --steps ./example-zitadel-init-steps.yaml \
    --masterkey "${ZITADEL_MASTERKEY}" --tlsMode disabled'
```
Previous was directly running the command from [docs](https://zitadel.com/docs/self-hosting/deploy/macos#run-zitadel), which caused same issue. I had to reinstall zitadel and clear all postgres databases and data in order to start fresh.

**Reply by fforootd** [reply]
Hm what error do you see in " but stops working after somedays of not using"?

**Reply by Susheelkaram** [reply]
@fforootd I am not seeing any error in either terminal logs or console UI. The command runs fine and I can open console to my local installation (http://localhost:8080/ui/console). But instead of taking in username password. 
It asks for username and then asks to activate user by entering code (assuming to be totp, which I didn't enable)  as in above screenshot.

**Reply by bmullan** [reply]
@Susheelkaram 
@fforootd 

I am having pretty similar problem as this 2024 question

I am self hosting netbird using zitadel

When I try to activate a User everything seems to indicate I am using  a 2factor ID which I never enabled?

I've tried hitting RESEND but still never receive any email w a code

any ideas

**Reply by hifabienne** [reply]
When you create a new user with no password and email not verified, the user will get an initial email to verify the email and/or add the password.
If you want to resend, make sure to use the resend init code instead of the email verification, as this is not the same process
